lib/timers.js

```diff
@@ -40,14 +40,17 @@ exports.active = function(item) {
     list = lists[msecs];
   } else {
     list = new Timer();
-    list.start(msecs, 0);
 
     L.init(list);
 
     lists[msecs] = list;
     list.msecs = msecs;
     list[kOnTimeout] = listOnTimeout;
   }
+  // Call start regardless whether list is new
+  // or not to prevent incorrect active_handles
+  // count. See https://github.com/nodejs/node-v0.x-archive/issues/25831.
```

Comment:
This new logic is definitely not correct.

If you do this you are at very high risk (almost 100%) of delaying existing, running timers. Read my extensive comments in https://github.com/nodejs/node/blob/7c9a691ee7d13ccf1c883e27fdb96eccc8b73d5e/lib/timers.js#L68-L78 on how this works if you don't believe me.

Perhaps it would work correctly if you did the following:

```js
list.start(Timer.now() - list._idlePrev._idleStart, 0);
```

If that worked, I would be willing to entertain changing the impl to that.